### PR TITLE
fix file group: use 0 instead of 'root'

### LINKED ISSUE #86: Fails on FreeBSD because gid 0 is wheel, not root
## Affected Puppet, Ruby, OS and module versions/distributions

- Puppet: 6.16.0
- Ruby: 2.6.6p146
- Distribution: FreeBSD 11.3
- Module version: 2.0.0

## How to reproduce (e.g Puppet code you use)

    cron::job { 'duplicity-inc':
      minute => fqdn_rand(60),
      hour => fqdn_rand(4),
      weekday => '1,2,3,4,5,6',
      user => 'root',
      command => '/usr/local/scripts/run-backup-to-b2 inc',
    }

## What are you seeing

```
Error: Could not find group root
Info: Unknown failure using insync_values? on type: File[job_duplicity-inc] / property: group to compare values ["root"] and 0
Error: /Stage[main]/Duplicity/Cron::Job[duplicity-inc]/File[job_duplicity-inc]/group: change from 'wheel' to 'root' failed: Could not find group root
```

## What behaviour did you expect instead

Successful run.

## Any additional information you'd like to impart

FreeBSD gid 0 is `wheel`, not `root`.

### PR DESCRIPTION
**Pull request by igalic**
#### Pull Request (PR) description

This should make it work on BSD systems.

#### This Pull Request (PR) fixes the following issues

Fixes #86

### CODE REVIEW

**Review by bastelfreak — APPROVED** [review]
can you also add the correct BSDs to metadata.json? :o

**Comment by bastelfreak** [comment]
labeling this enhancement since this module currently does not claim to be BSD compatible.

File: metadata.json [hunk 1]
```diff
@@ -45,6 +45,12 @@
       "operatingsystemrelease": [
         "12"
       ]
+    },
+    {
+      "operatingsystem": "FreeBSD"
```
Comment (bastelfreak) [inline metadata.json:50]: mhm, sorry to nitpick here :( Can you add the Non-EOL version where this is supposed to work? (same for NetBSD)

Comment (igalic) [reply metadata.json:50]: i just went with what Gentoo does ;)

**Review by bastelfreak — APPROVED** [review]

**Comment by igalic** [comment]
@Flupsy, you happy with this?

**Comment by Flupsy** [comment]
> 
> 
> @Flupsy, you happy with this?

Very!  Thanks so much for the quick turnaround.

**Pull request merged**